extide/gradle/apichanges.xml

```diff
@@ -83,6 +83,21 @@ is the proper place.
     <!-- ACTUAL CHANGES BEGIN HERE: -->
 
     <changes>
+        <change id="gradle-setting-deprecation">
+            <api name="general"/>
+            <summary>Some Gradle Settings Removed from Use</summary>
+            <version major="2" minor="36"/>
+            <date day="2" month="10" year="2023"/>
+            <author login="lkishalmi"/>
+            <compatibility semantic="compatible" />
```

Comment:
IMHO semantic should be `incompatible`: code that hopes the setter (setting) has some effect may break.